modules/tinymce/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added support for alpha list numbering to the `lists` plugin #TINY-6891
 - The editor resize handle can now be controlled using the keyboard #TINY-4823
 
+### Changed
+- The "H Align" and "V Align" input labels in the Cell Properties dialog to be "Horizontal align" and "Vertical align" respectively #TINY-7285
```

Comment:
```suggestion
- The "H Align" and "V Align" input labels in the Table Cell Properties dialog to be "Horizontal align" and "Vertical align" respectively #TINY-7285
```

Reply:
Done as a commit, since needed to do another change too.